### ISSUE TITLE
Fails to initialize - remote.ipAddress

### ISSUE THREAD

**Issue by andymikulski** [feross/simple-peer#66]
Running `webtorrent-hybrid` in node, attempting to seed an instant.io - generated/seeded magnet produces the following error:

```
self.remoteAddress = remote.ipAddress
                           ^
TypeError: Cannot read property 'ipAddress' of undefined at setActiveCandidates
```

**Comment by feross** [comment]
What browser and version?

**Comment by andymikulski** [comment]
Forgive me! Should have included that info:

Chrome 48.0.2564.109 (64-bit)
OSX v10.10.4
Node v4.2.2

**Comment by feross** [comment]
Does it happen every time? Or just sometimes?

It would help me immensely if you could add this line:

```js
console.log(item)
```

directly above this line: https://github.com/feross/simple-peer/blob/master/index.js#L421

And paste the logs.

**Comment by substack** [comment]
```
item= { googChannelId: 'Channel-data-1',
  bytesSent: '2779',
  packetsDiscardedOnSend: '0',
  packetsSent: '7',
  bytesReceived: '2108',
  googWritable: 'true',
  googReadable: 'true',
  googActiveConnection: 'true',
  localCandidateId: 'Cand-UNmc8fA+',
  remoteCandidateId: 'Cand-',
  googLocalAddress: '192.168.3.149:47636',
  googRemoteAddress: '192.168.3.149:40563',
  googRtt: '1687',
  googTransportType: 'udp',
  googLocalCandidateType: 'local',
  googRemoteCandidateType: 'local',
  id: undefined,
  type: 'googCandidatePair',
  timestamp: 1455672192253.142 }
/home/substack/projects/swarmbot/node_modules/simple-peer/index.js:420
      self.remoteAddress = remote.ipAddress
```

**Comment by substack** [comment]
and remoteCandidates looks like this:

```
remoteCandidates= { undefined: 
   { ipAddress: '192.168.3.149',
     portNumber: '57257',
     priority: '1853628159',
     candidateType: 'peerreflexive',
     transport: 'tcp',
     id: undefined,
     type: 'remotecandidate',
     timestamp: 1455672276531.792 } }
```

**Comment by feross** [comment]
@substack `wrtc` is doing the wrong thing here, clearly. The `id` property is `undefined` in multiple places. Also, the `remoteCandidateId` looks incomplete (`'Cand-'`). But, it looks like we can get the same info from `googLocalAddress` and `googRemoteAddress` as a fallback.

@andymikulski Can't reproduce this in Chrome, but I'll make this code more defensive all the same. Better to have `peer.remoteAddress`, etc. be `undefined` than to crash like this.

**Comment by andymikulski** [comment]
Happens every time.

[Here's my log](https://gist.github.com/andymikulski/39a1d5881822ad2bf3b1), gist'd for brevity

**Comment by feross** [comment]
I don't understand why `id` is `undefined` in both these examples. Asking for help on twitter: https://twitter.com/feross/status/699769571247419392

In any case, will be defensive against this.

**Comment by feross** [comment]
Released a fix as 6.0.1. @substack @andymikulski Please confirm that it works for you!
